test/fixtures/project_eslint/src/lib/func.ts

```diff
@@ -0,0 +1,3 @@
+// This code should trigger this error
+// Array type using 'Array<string>' is forbidden. Use 'string[]' instead.eslint(@typescript-eslint/array-type)
+export const arrayOfStringTriggeringError: Array<string> = [];
```

Comment:
I think we are missing test that lint js + ts files

Reply:
Will do